### DISCUSSION TITLE
Software evolution

### DISCUSSION THREAD

**Discussion by Raf-sns** [Raf-sns/Placido-Shop#3] [Ideas]
Let's discuss the evolutions you would like to see integrated into Placido-Shop.

**Comment by Lamudesign** [comment]
Bonjour cher Raf ! Y a t-il une option pour mettre le site en maintenance ? Merci !

**Reply by Raf-sns** [reply]
Hello Mumu!
Unfortunately there is no maintenance mode on Placido-Shop, how do you imagine what a maintenance mode would produce? 
I might implement it in the upcoming new version.

**Reply by Lamudesign** [reply]
Juste une page qui dit que le site est en maintenance, avec le logo et un lien pour rediriger ailleurs éventuellement...
